wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,286 @@
+
+
+## Mappools
+
+### Qualifiers
+
+![Qualifiers Mappool](Qaulifiers.jpg)
+
+- NoMod
+  - [Ceui - COLORFUL DAYS!!](https://osu.ppy.sh/b/203315) [Insane]
+  - [Mastermind(xi+nora2r) - Dreadnought](https://osu.ppy.sh/b/813331) [Another]
+  - [Function Phantom - Euclid](https://osu.ppy.sh/b/971086) [toybot's Insane]	
+- DoubleTime
+  - [Duca - Clover](https://osu.ppy.sh/b/157856) [Collab]	
+  - [Icon For Hire - Make A Move](https://osu.ppy.sh/b/1077369) [Hard]		
+- Hidden  
+  - [Dreamcatcher - Sleep-walking](https://osu.ppy.sh/b/1370703) [Insane]	
+  - [yuiko - petit bonheur](https://osu.ppy.sh/b/1327955) [Insane]	
+- Hardrock
+  - [Otokaze - Karen (Short Ver.)](https://osu.ppy.sh/b/154357) [Hanabi]		
+  - [ALiCE'S EMOTiON - Dark Flight Dreamer](https://osu.ppy.sh/b/763294) [Twaoi's Insane]			
+### Group Stage
+
+![Group Stage Mappool](Group.jpg)
+
+- NoMod
+  - [GARNiDELiA - ambiguous](https://osu.ppy.sh/b/680886) [Insane]
+  - [Mind Vortex - Arc](https://osu.ppy.sh/b/239037) [NSane]
+  - [sakuzyo - VALLISTA](https://osu.ppy.sh/b/127313) [Another]
+  - [B-ko(Cv:Touyama Nao) - Nisemono Chuuihou](https://osu.ppy.sh/b/1253351) [Insane]
+  - [Mediks - Doomsday](https://osu.ppy.sh/b/1290316) [Insane]
+  - [Dark PHOENiX - Taketori Hishou](https://osu.ppy.sh/b/86324) [Extra]
+- Double Time
+  - [Two Steps From Hell - Star Sky](https://osu.ppy.sh/b/1270965) [Unstoppable]
+  - [Petit Rabbit's - No Poi!(nenpulse bootleg remix)](https://osu.ppy.sh/b/937188) [YakiBoss's Hard]
+  - [Chata - Koi wo Shiyouyo](https://osu.ppy.sh/b/81200) [Insane]
+- Hidden
+  - [Kano - Yuudachi no Ribbon](https://osu.ppy.sh/b/1078420) [Dailycare's Insane]
+  - [Renard - Terminal](https://osu.ppy.sh/b/270363) [Ends' Insane]
+  - [Ne;on & Haiji - Haruka Tooki Gensoukyo - Dream of fantasy](https://osu.ppy.sh/b/106475) [lkp's Lunatic]
+- Hardrock
+  - [Rchetype - V Is For](https://osu.ppy.sh/b/290919) [Insane]
+  - [MOMOIRO CLOVER Z - SARABA ITOSHIKI KANASHIMI TACHIYO](https://osu.ppy.sh/b/507098) [Master]
+  - [Sota Fujimori - Move That Body -Extended Mix-](https://osu.ppy.sh/b/547229) [RLC's Insane]
+- Free Mod
+  - [Kozato Snow - Izayoi Sakura](https://osu.ppy.sh/b/86352) [Insane]
+  - [iojjj - Deus Ex Machina](https://osu.ppy.sh/b/688907) [EXTRA]
+  - [Mutsuhiko Izumi - Red Goose](https://osu.ppy.sh/b/144028) [Another]
+- Tiebreaker
+  - [Demetori - The Grimoire of Alice ~ Alice in Wonderland](https://osu.ppy.sh/b/628369) [Lolis]
+
+### Round Of 16
+
+![Round Of 16 Mappool](Round.jpg)
+
+ - NoMod
+  - [Eiko Shimamiya - Higurashi no Naku Koro ni](https://osu.ppy.sh/b/738088) [Extra]
+  - [EYKHA (CV : Sawashiro Miyuki) LOU (CV : Uchida Maaya) MERYL (CV : Terui Haruka) - Delta Decision](https://osu.ppy.sh/b/1097612) [Linada's Extra]  	
+  - [Suzaku - Anisakis -somatic mutation type''Forza''(https://osu.ppy.sh/b/199593) [Ultra]
```

Comment:
I'm not sure what happened here, but it should probably look like this
`[Suzaku - Anisakis -somatic mutation type''Forza''-](https://osu.ppy.sh/b/199593)`

Reply:
fixed